joint_trajectory_controller/CMakeLists.txt

```diff
@@ -74,6 +79,16 @@ if(BUILD_TESTING)
     "test_robot_hardware"
     "trajectory_msgs"
   )
+
+  ament_add_gtest(
+    test_load_joint_trajectory_controller
+    test/test_load_joint_trajectory_controller.cpp
+  )
+  target_include_directories(test_load_joint_trajectory_controller PRIVATE include)
+  ament_target_dependencies(test_load_joint_trajectory_controller
+    controller_manager
+    test_robot_hardware
```

Comment:
I got include errors when building this with a tweaked example code shown in the ros2_control github. Not sure how it passed the automated tests?

Anyway, adding the following to ament_target_dependencies fixes it.
`"controller_interface"`

Reply:
The code here does not on it's own impose any dependencies on that library, perhaps one of the dependencies included does not properly link/export it and your local setup produced a broken tree due to it.

Reply:
@ddengster Do you have this commit on your `ros2_control`? https://github.com/ros-controls/ros2_control/pull/58

Reply:
> @ddengster Do you have this commit on your `ros2_control`? [ros-controls/ros2_control#58](https://github.com/ros-controls/ros2_control/pull/58)

Ah that fixed it, thanks.

Reply:
If you guys can confirm it is a problem, please open new issue to track it.